index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <body>
```

Comment:
The body must contain all the visible elements in the web page, but it mustn't be within the head tag.

Reply:
- [ ] It will be better to give your change request politely, as suggestions instead of orders.
For example: 
"The body must contain all HTML elements, not in the `head` tag, Kindly, try to solve this issue 😄"

Reply:
🌝 The `<body>` must contain all the visual elements of the page, not the `<head>` 👀 .
Please go and try to solve this problem 🌚 .